### PR TITLE
Change the default values of hostname and protocol to current values of the page

### PR DESCRIPTION
**Pull request by ArnoSaine**
By default it's possible to access the server using other hostnames as well, like from a virtual machine by IP.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ test('children trigger notification', function(t) {
 
 test('autoloadJs hostname defaults to localhost', function(t) {
```
Comment (statianzo) [inline test.js]: This test description needs to be updated to match the logic of the test

File: README.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ Add a script tag to your page pointed at the livereload server
 
 ## Options
 
-- `protocol` - (Default: `http`) Protocol for livereload `<script>` src attribute value
+- `protocol` - (Default: protocol of the page, either `http` or `https`) Protocol for livereload `<script>` src attribute value
 - `port` - (Default: 35729) The desired port for the livereload server
-- `hostname` - (Default: `localhost`) The desired hostname for the appended
+- `hostname` - (Default: hostname of the page, like `localhost` or `10.0.2.2`) The desired hostname for the appended
```
Comment (statianzo) [inline README.md]: Thanks for updating the docs 👍

File: test.js (hunk 1 above)
Comment (ArnoSaine) [reply test.js]: Oh, that's right.

**Comment by statianzo** [comment]
Thanks for the contribution!

**Comment by statianzo** [comment]
Pushed this as 1.0 because it's a breaking change.

**Comment by ArnoSaine** [comment]
Great. Thanks!

**Pull request merged**